src/main/java/org/opensearch/ad/transport/EntityResultTransportAction.java

```diff
@@ -75,9 +76,10 @@
     private CheckpointDao checkpointDao;
     private CacheProvider cache;
     private final NodeStateManager stateManager;
-    private final int coolDownMinutes;
+    private int coolDownMinutes;
```

Comment:
Will the update consumer work by using "private int" ? I think we should use "private volatile int"

Reply:
Thanks! will update in another commit.